superset-frontend/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -222,6 +224,7 @@ class SqlEditor extends React.PureComponent {
 
   componentWillUnmount() {
     window.removeEventListener('resize', this.handleWindowResize);
+    window.removeEventListener('beforeunload', this.onBeforeUnload.bind(this));
```

Comment:
don't think you need the bind here, since this has already been bound when the method was declared.

Reply:
thanks, fixed by efb86e51af1959de182faf486a6877164ebd933f

Reply:
Oh, true. And also I think it actually won't work with `.bind` since that returns a new function.